src/position-controller.cpp

```diff
@@ -76,15 +75,17 @@ namespace dynamicgraph
 
         /* Commands. */
         addCommand("init",
-                   makeCommandVoid1(*this, &PositionController::init,
-                                    docCommandVoid1("Initialize the entity.",
-                                                    "Time period in seconds (double)")));
+                   makeCommandVoid2(*this, &PositionController::init,
+                                    docCommandVoid2("Initialize the entity.",
+                                                    "Time period in seconds (double)",
+						    "Size of the state vector (double)")));
```

Comment:
I think "size of the state vector" is an ambiguous description. Maybe "number of joints (not including floating base)" is better.